File: ios/LinearSearch.swift
//
//  LinearSearch.swift
//  kata
//
//  Created by Kévin La Rosa on 21/07/2023.
//

import Foundation

public struct LinearSearch {
  public static func search(orderedArray: [Int], numberSearch: Int) -> Bool {
    return orderedArray.first { $0 == numberSearch } != nil
  }
}

File: android/app/src/main/java/com/kata/LinearSearch.kt
package com.kata

class LinearSearch {
    companion object {
        fun search (orderedArray: Array<Int>, numberSearch: Int): Boolean {
            for (element in orderedArray)
            {
                if (element == numberSearch) {
                    return true
                }
            }
            return false
        }
    }
}